packages/expect/src/__tests__/extend.test.ts

```diff
@@ -109,7 +109,7 @@ it('exposes an equality function to custom matchers', () => {
   jestExpect.extend({
     toBeOne() {
       expect(this.equals).toBe(equals);
-      return {pass: !!this.equals(1, 1)};
+      return {pass: Boolean(this.equals(1, 1))};
```

Comment:
I would expect from something that is named like `equals` to already return boolean :)

Reply:
then what should i do here?

Reply:
`this.equals` does return a boolean, so just remove the `!!`

Reply:
sure